source/_integrations/nest.markdown

```diff
@@ -38,15 +38,15 @@ Cameras and doorbells use [Automation and device triggers](#automation-and-devic
 
 You are in control of the information and capabilities exposed to Home Assistant. You can authorize a single device, multiple devices, or different levels of functionality such as motion events, live streams, for any particular device. The integration is flexible enough to adapt based on what you allow.
 
+# Prerequisites
+
 {% caution %}
```

Comment:
```suggestion
```
I think the text boxes could be removed here. The Prerequisites block already indicates that these topics need to be checked or done before anything else.

Reply:
In theory i agree with you... However, users are typically pretty *furious* over missing this. It has previously been a little subtle and users miss it. There are probably tens of reports on this topic in this repo or in the core repo...

If you still feel this should be changed in the name of consistency happy to make the change but just want to give the context.

Reply:
Eh, let's go for it and see what happens...

Reply:
I don't have any preference on this, will use whatever should be used. I did go ahead and apply your suggestions though. The Prerequisites heading may be clear enough so i'm open to try it.